bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/tools/WaitForTopicValue.java

```diff
@@ -44,20 +45,24 @@
     public WaitForTopicValue(MqttBrokerConnection connection, String topic)
             throws InterruptedException, ExecutionException {
```

Comment:
I don't see anywhere this would be thrown. It should be safe to remove these exceptions.